integration_tests/tests/temporary-accommodation/manage/report.cy.ts

```diff
@@ -68,43 +40,51 @@ context('Report', () => {
     cy.task('stubBookingReportReferenceData')
     const page = BookingReportNewPage.visit()
 
-    // And I submit the form without choosing a region out the form
-    cy.task('stubBookingReportError', 'some-data')
+    // And I fill out the form
+    cy.then(function _() {
+      const probationRegion = this.actingUserProbationRegion
+      cy.task('stubBookingReportError', { data: 'some-data', probationRegionId: probationRegion.id })
+    })
 
     page.expectDownload()
     page.clickSubmit()
 
     cy.get('h1').contains('Internal Server Error')
   })
 
-  it('allows me to download a booking report for a region', () => {
+  it("allows me to download a booking report for the acting user's region", () => {
```

Comment:
I think we should stick with single quotes - seems to be what's used throughout the rest of the file

Reply:
I agree but the linter doesn't! The linter prefers double quotes, rather than escaping the single quote within the string